lib/simplekiq/orchestration_executor.rb

```diff
@@ -13,9 +13,7 @@ def self.execute(args:, job:, workflow:)
       Simplekiq.auto_define_callbacks(orchestration_batch, args: args, job: job)
 
       orchestration_batch.jobs do
-        Simplekiq::BatchTrackerJob.perform_async(job.class.name, orchestration_batch.bid, args)
-
-        new.run_step(workflow, 0)
+        Simplekiq::KickoffJob.perform_async(workflow)
```

Comment:
This appears to potentially break the `on_complete` and `on_success` callbacks for an `OrchestrationJob`, as they now run immediately after the workflow begins